addons/im_livechat/tests/test_session_history.py

```diff
@@ -18,6 +18,8 @@ def test_session_history_navigation_back_and_forth(self):
         })
         channel = self.env["discuss.channel"].browse(data["discuss.channel"][0]["id"])
         channel.with_user(operator).message_post(body="Hello, how can I help you?")
+        self.env.cr.precommit.run()
+        self.env["bus.bus"].sudo().search([]).unlink()
```

Comment:
_reset_bus available in this class?

Reply:
created a dedicated bus common class to provide `_reset_bus` without the whole mail setup.